src/services/brandingService.ts

```diff
@@ -19,31 +17,3 @@ export const removeCredentialBranding = async (args: IRemoveCredentialBrandingAr
   debug(`removeCredentialBranding(${JSON.stringify(args)})...`);
   return ibRemoveCredentialBranding(args);
 };
-
-export const selectAppLocaleBranding = async (
```

Comment:
Did this get moved to ui-components or is it just rmeoved?

Reply:
yeah this is moved to the `credential-branding` module